src/app/schemas/setting_autocomplete.py

```diff
@@ -21,3 +24,34 @@ class Meta:
     sheet = fields.Str(required=True)
     from_row = fields.Str(required=True, data_key="fromRow")
     to_row = fields.Str(required=True, data_key="toRow")
+
+    @validates_schema
+    # pylint:disable=no-self-use
+    def validate_data_url(self, data, **kwargs):
+        """
+        Validates url, which must be docs.google.com
+        :param data:
+        :param kwargs:
+        :return:
+        """
+        parsed_url = urlparse(data.get('data_url'))
+        if parsed_url.netloc != 'docs.google.com':
+            raise ValidationError('Wrong URL entered')
```

Comment:
Exactly the same method as in schemas/forms.py

Reply:
fixed